src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsresettlementpassportapi/integration/AppointmentsIntegrationTest.kt

```diff
@@ -157,6 +157,23 @@ class AppointmentsIntegrationTest : IntegrationTestBase() {
       .expectBody().json(expectedOutput)
   }
 
+  @Test
+  @Sql("classpath:testdata/sql/seed-pathway-statuses-6.sql")
+  fun `Get appointment by id`() {
```

Comment:
Might be worth adding a couple of tests for the Unauthorised and Forbidden cases to ensure that's set up correctly for this endpoint?